InventoryInStorePickupSalesAdminUi/view/adminhtml/layout/sales_order_view.xml

```diff
@@ -8,7 +8,7 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceContainer name="content">
-            <block class="Magento\InventoryInStorePickupSalesAdminUi\Block\Adminhtml\Order\View\ReadyForPickup" name="sales_order_ready_for_pickup"/>
```

Comment:
Is `Magento\InventoryInStorePickupSalesAdminUi\Block\Adminhtml\Order\View\ReadyForPickup` should be removed after such changes or it's used somewhere else?

Reply:
@sidolov, yes, looks like I've forgotten to remove it. Done.